scripts/build-site.sh

```diff
@@ -35,7 +35,4 @@ yarn run tsc --outFile ${JS_BUNDLE}
 printf "Compiling Sass and running PostCSS...\n\n"
 yarn run --silent node-sass assets/sass/styles.scss | yarn run postcss --config assets/config --output ${CSS_BUNDLE}
 
-printf "Building the search index...\n\n"
-make build_search_index
```

Comment:
We can also remove the `build_search_index` target from the `Makefile`:

https://github.com/pulumi/docs/blob/26a033fb30a38ebc19eeac0b395ad381c597b713/Makefile#L48-L52